### PR TITLE
Added docs on deployment

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Upload deploy-cluster-CN, fix typo in deploy-cluster, add 

> meta_server_addrs: List of meta server addresses, the format looks like ip1:port1, ip2:port2, ip3:port3

to get-started CN and EN.

### CODE REVIEW

File: docs/manual-CN/deploy-cluster.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+ `Ubuntu`系统：
+
+```
+dpkg -i nebula-{VERSION}.{SYSTEM_VERSION}.amd64.deb
+```
+
+配置文件默认目录为`/usr/local/nebula/etc`，请更改`meta_server_addrs`，配置 Meta Server 的地址。
+
+In order to enable multi copy Meta service, you should set the meta addresses split by comma into `meta_server_addrs`.
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: ？？

File: docs/manual-CN/deploy-cluster.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+启动多副本 Meta 服务需将多个地址配置到`meta_server_addrs`，地址间需使用逗号隔开。
+
+使用`data_path`设置`Meta`的基础存储路径。
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: `Meta` 和 `Storage` 底层存储路径？

File: docs/manual-CN/deploy-cluster.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+***
+
+#### 启动 Nebula 集群
+
+
+目前，nebula 集群由`scripts/services.sh`管理，可使用此脚本`start`，`stop`或 `restart`重启集群。
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: 管理 =》 运维吧

File: docs/manual-CN/deploy-cluster.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+#### 配置引用
+
+**Meta Service** 支持如下配置属性：
+
+属性名               | 默认值            | 说明
+--------------------------- | ------------------------ | -----------
+`port`                      | 45500                    | Meta daemon 监听端口
+`reuse_port`                | true                     | 开启 SO_REUSEPORT 选项
+`data_path`                 | ""                       | 根数据路径
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: 数据目录

File: docs/manual-CN/deploy-cluster.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+`peers`                     | ""                       | 一系列由逗号隔开的IP地址，IP数与副本数相等，如果为空，则表明副本数为1
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: peers to meta_server_addrs

File: docs/manual-CN/deploy-cluster.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+`meta_ingest_thread_num`    | 3.                       | Meta daemon的 ingest 线程数
+
+**Storage Service** 支持如下配置属性：
+
+属性名                       | 默认值              | 说明
+----------------------------------- | -------------------------- | -----------
+`port`                              | 44500                      | Storage daemon 监听端口
+`reuse_port`                        | true                       | 开启 SO_REUSEPORT 选项
+`data_path`                         | ""                         | 根数据路径，多条路径由逗号隔开，对 rocksdb 引擎，一个路径为一个实例
+`local_ip`                          | ""                         |  IP 地址和监听端口用于此服务器
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: 和 `port` 一起标识 host ?

File: docs/manual-CN/deploy-cluster.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+`meta_server_addrs`                 | ""                         | meta server 地址列表，格式为 ip1:port1, ip2:port2, ip3:port3
+`store_type`                        | "nebula"                   | storage daemon 使用的 KVStore 类型，可选类型为 \"nebula\"和\"hbase\"
+`num_io_threads`                    | 16                         | IO 线程数
+`storage_http_thread_num`           | 3                          | storage daemon 的 http 线程数
+`num_worker_threads`                | 32                         | worker 数
+`engine_type`                       | rocksdb                    | rocksdb, memory...
+`custom_filter_interval_secs`       | 24 * 3600                  | 触发自定义压缩间隔
+`num_workers`                       | 4                          | worker 线程数
+`rocksdb_disable_wal`               | false                      |  禁用 rockdb 的 WAL
+`rocksdb_db_options`                | ""                         |  DBOptions，每个选项以 <option_name>:<option_value> 格式给出，以.隔开
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: 以 `.` 分隔

File: docs/manual-CN/deploy-cluster.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+`rocksdb_column_family_options`     | ""                         | ColumnFamilyOptions，每个选项以 <option_name>:<option_value> 格式给出，以.隔开
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: ditto

File: docs/manual-CN/deploy-cluster.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+`rocksdb_block_based_table_options` | ""                         | BlockBasedTableOptions，每个选项以<option_name>:<option_value> 格式给出，以.隔开
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: ditto

File: docs/manual-CN/deploy-cluster.md [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+`batch_size`                        | 4 * 1024                   | 单个批处理的默认保留字节
+`block_cache`                       | 4                          | BlockBasedTable:block_cache : MB
+`download_thread_num`               | 3                          | 下载线程数
+`min_vertices_per_bucket`           | 3                          | 一个bucket中最小节点数
+`max_appendlog_batch_size`          | 128                        | 每个appendLog批请求的最大log数
+`max_outstanding_requests`          | 1024                       | outstanding appendLog 请求最大数
+`raft_rpc_timeout_ms`               | 500                        | raft 客户端 RPC 超时时长
+`accept_log_append_during_pulling`  | false                      |  pull snapshot 过程中不接受新log
+`raft_heartbeat_interval_secs`      | 5                          | 每次心跳间隔时长
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: 每次心跳间隔时长, 单位秒

File: docs/manual-CN/deploy-cluster.md [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+
+**Console** 支持如下配置属性：
+
+属性名            | 默认值 | 说明
+------------------------ | ------------- | -----------
+`addr`                   | "127.0.0.1"   | Nebula daemon IP 地址
+`port`                   | 0             | Nebula daemon 监听端口
+`u`                      | ""            | 用于身份验证的用户名
+`p`                      | ""            | 用于身份验证的密码
+`enable_history`         | false         | 不强制存储命令大历史
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: 是否强制保存command历史

File: docs/manual-CN/deploy-cluster.md [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+`server_conn_timeout_ms` | 1000          | 连接超时时长
```
Comment (jude-zhu) [inline docs/manual-CN/deploy-cluster.md]: 加单位

File: docs/deploy-cluster.md [hunk 13]
```diff
@@ -146,4 +146,4 @@ Property Name            | Default Value | Description
 `enable_history`         | false         | Whether to force saving the command history.
 `server_conn_timeout_ms` | 1000          | Connection timeout in milliseconds.
 
-
+**Note:** Please make sure the ports are not blocked by the firewall during configuration.
```
Comment (darionyaphet) [inline docs/deploy-cluster.md]: nice

File: docs/manual-CN/deploy-cluster.md [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+`pid_file`                          | "pids/nebula-storaged.pid" | 存储进程 ID 的文件
+`meta_server_addrs`                 | ""                         | meta server 地址列表，格式为 ip1:port1, ip2:port2, ip3:port3
+`store_type`                        | "nebula"                   | storage daemon 使用的 KVStore 类型，可选类型为 \"nebula\"和\"hbase\"
+`num_io_threads`                    | 16                         | IO 线程数
+`storage_http_thread_num`           | 3                          | storage daemon 的 http 线程数
+`num_worker_threads`                | 32                         | worker 数
+`engine_type`                       | rocksdb                    | rocksdb, memory...
+`custom_filter_interval_secs`       | 24 * 3600                  | 触发自定义压缩间隔
+`num_workers`                       | 4                          | worker 线程数
+`rocksdb_disable_wal`               | false                      |  禁用 rockdb 的 WAL
```
Comment (darionyaphet) [inline docs/manual-CN/deploy-cluster.md]: alignment

File: docs/manual-CN/get-started.md [hunk 15]
```diff
@@ -145,6 +145,7 @@ nebula 遵循 c++14 标准，依赖第三方库：
    - port: 端口号
    - ws_http_port: metaservice HTTP服务端口号
    - ws_h2_port: metaservice HTTP2服务端口号
+   - meta_server_addrs: meta server 地址列表，格式为 ip1:port1, ip2:port2, ip3:port3
```
Comment (darionyaphet) [inline docs/manual-CN/get-started.md]: remove the extra space

File: docs/deploy-cluster.md (hunk 13 above)
Comment (Amber1990Zhang) [reply docs/deploy-cluster.md]: @jude-zhu thanks jude

File: docs/manual-CN/deploy-cluster.md (hunk 14 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/deploy-cluster.md]: Have checked the review, the alignment is good, thank for your remind.

File: docs/manual-CN/deploy-cluster.md [hunk 16]
```diff
@@ -0,0 +1,146 @@
...
+`data_path`                 | ""                       | 根数据目录
+`meta_server_addrs`                     | ""                       | 一系列由逗号分隔的IP地址，IP数与副本数相等，如果为空，则表明副本数为1
+`local_ip`                  | ""                       | 指定本地IP  NetworkUtils::getLocalIP.
+`num_io_threads`            | 16                       | IO 线程数
+`meta_http_thread_num`      | 3                        | meta daemon 的 http 线程数
+`num_worker_threads`        | 32                       | worker 数
+`part_man_type`             | memory                   | memory，meta
+`pid_file`                  | "pids/nebula-metad.pid"  | 存储进程 ID 的文件
+`daemonize`                 | true                     | 作为 daemon 进程运行
+`cluster_id`                | 0                        | 每个集群都惟一的 ID
```
Comment (dangleptr) [inline docs/manual-CN/deploy-cluster.md]: Do we have this configuration?

File: docs/manual-CN/deploy-cluster.md [hunk 17]
```diff
@@ -0,0 +1,146 @@
...
+`putTryNum`                 | 10                       | 尝试生成集群 ID 的次数
```
Comment (dangleptr) [inline docs/manual-CN/deploy-cluster.md]: Remove it,  it is stale now.

File: docs/manual-CN/deploy-cluster.md [hunk 18]
```diff
@@ -0,0 +1,146 @@
...
+`load_config_interval_secs` | 2 * 60                   | 加载配置间隔
+`meta_ingest_thread_num`    | 3.                       | Meta daemon的 ingest 线程数
```
Comment (dangleptr) [inline docs/manual-CN/deploy-cluster.md]: Maybe it has another name now,  please confirm it with @darionyaphet

File: docs/manual-CN/deploy-cluster.md [hunk 19]
```diff
@@ -0,0 +1,146 @@
...
+
+#### 配置引用
+
+**Meta Service** 支持如下配置属性：
+
+属性名               | 默认值            | 说明
+--------------------------- | ------------------------ | -----------
+`port`                      | 45500                    | Meta daemon 监听端口
+`reuse_port`                | true                     | 开启 SO_REUSEPORT 选项
+`data_path`                 | ""                       | 根数据目录
```
Comment (dangleptr) [inline docs/manual-CN/deploy-cluster.md]: We should tell our users,   data_path could not support multi paths like storage service for metad

File: docs/manual-CN/how-to-build.md [hunk 20]
```diff
@@ -136,35 +136,35 @@ bash> sudo make install
...
 
     通过 yum install
-    
+
     ```
     bash> sudo yum -y install libtool autoconf autoconf-archive automake perl-WWW-Curl perl-YAML perl-CGI glibc-devel libstdc++-static maven java-1.8.0-openjdk
     ```
-    
+
     和通过 vesoft offer
```
Comment (whitewum) [inline docs/manual-CN/how-to-build.md]: watch here!

File: docs/manual-CN/how-to-build.md [hunk 21]
```diff
@@ -204,45 +204,45 @@ bash> sudo make install
...
 
     通过 apt-get install
-    
+
     ```
     bash> sudo apt-get -y install gcc-multilib libtool autoconf autoconf-archive automake python maven java-1.8.0-openjdk
     ```
-    
+
     通过 vesoft offer
```
Comment (whitewum) [inline docs/manual-CN/how-to-build.md]: aha?

**Comment by whitewum** [comment]
two minor errors.

File: docs/manual-CN/how-to-build.md (hunk 21 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/how-to-build.md]: changed to '通过 vesoft 提供的包'， sorry I just updated the online version only.

File: docs/manual-CN/deploy-cluster.md (hunk 16 above)
Comment (whitewum) [reply docs/manual-CN/deploy-cluster.md]: > Do we have this configuration?

who can confirm this? @dangleptr @critical27

File: docs/manual-CN/how-to-build.md [hunk 22]
```diff
@@ -136,35 +136,35 @@ bash> sudo make install
 - 安装依赖模块
 
     通过 yum install
-    
+
     ```
     bash> sudo yum -y install libtool autoconf autoconf-archive automake perl-WWW-Curl perl-YAML perl-CGI glibc-devel libstdc++-static maven java-1.8.0-openjdk
     ```
-    
+
```
Comment (whitewum) [inline docs/manual-CN/how-to-build.md]: offer ??

File: docs/deploy-cluster.md [hunk 23]
```diff
@@ -62,7 +62,6 @@ Property Name               | Default Value            | Description
 `pid_file`                  | "pids/nebula-metad.pid"  | File to hold the process id.
 `daemonize`                 | true                     | Whether run as a daemon process.
 `cluster_id`                | 0                        | A unique id for each cluster.
```
Comment (dangleptr) [inline docs/deploy-cluster.md]: IIRC,  "cluster_id" is removed again,  we have one flag "cluster_id_path" which is used to save clusterId on this path.

File: docs/deploy-cluster.md [hunk 24]
```diff
@@ -52,8 +52,8 @@ Property Name               | Default Value            | Description
 --------------------------- | ------------------------ | -----------
 `port`                      | 45500                    | Meta daemon listening port.
 `reuse_port`                | true                     | Whether to turn on the SO_REUSEPORT option.
-`data_path`                 | ""                       | Root data path.
-`peers`                     | ""                       | It is a list of IPs split by comma, the ips number equals replica number. If empty, it means replica is 1.
+`data_path`                 | ""                       | Root data path. Multi-path is not supported
```
Comment (dangleptr) [inline docs/deploy-cluster.md]: We don't support multi-path for meta service.

**Comment by dutor** [comment]
Please resolve the conflicts first.

**Pull request merged**